packages/commandkit/bin/common.mjs

```diff
@@ -72,16 +72,23 @@ const possibleFileNames = [
 export async function findCommandKitConfig(src) {
     const cwd = process.cwd();
     const locations = src ? [join(cwd, src)] : possibleFileNames.map((name) => join(cwd, name));
+    const foundConfigFiles = [];
 
     for (const location of locations) {
         try {
-            return await loadConfigInner(location);
+            foundConfigFiles.push(location);
```

Comment:
this is completely wrong, you are just creating new array that exactly equals `locations` array.